lib/collins/cli/find.rb

```diff
@@ -40,6 +40,7 @@ def parse!(argv = ARGV)
         opts.banner = "Usage: #{PROG_NAME} [options] [hostnamepattern]"
         opts.separator "Query options:"
         opts.on('-t','--tag TAG[,...]',Array, "Assets with tag[s] TAG") {|v| search_attrs[:tag] = v}
+        opts.on('-Z','--remote',"Query remote datacenters for asset") {|v| search_attrs[:remoteLookup] = v}
```

Comment:
can you set `search_attrs[:remoteLookup] = false` as default, or are you comfortable relying on the API's defaults?

Also, maybe make the long flag map to the attribute in the API? `--remote-lookup`?